sdk/communication/Azure.Communication.MediaComposition/tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.MediaComposition.Models;
+
+namespace Azure.Communication.MediaComposition.Tests.samples
+{
+    /// <summary>
+    /// Basic Azure Communication.MediaComposition samples.
+    /// </summary>
+    public partial class Sample1_MediaCompositionClient
+    {
+        // TODO: Update the samples to tests when the E2E flow is working
+        private const string mediaCompositionId = "2x2Grid";
+        public async Task CreateMediaCompositionAsync()
+        {
+            var mediaCompositionClient = CreateMediaCompositionClient();
+
+            var layout = new MediaCompositionLayout()
+            {
+                Resolution = new LayoutResolution(1920, 1080),
+                Grid = new GridLayoutOptions(
+                    rows: 2,
+                    columns: 2,
+                    new List<List<string>>
+                    {
+                        new List<string>
+                        {
+                            "jill", "jack"
+                        },
+                        new List<string>
+                        {
+                            "jane", "jerry"
+                        }
+                    })
+            };
+
+            var inputs = new Dictionary<string, MediaInput>()
+            {
+                {
+                    "jill",
+                    new MediaInput() {
+                        Participant = new ParticipantInput(
+                        id: new CommunicationIdentifierModel() { MicrosoftTeamsUser = new MicrosoftTeamsUserIdentifierModel("f3ba9014-6dca-4456-8ec0-fa03cfa2b7b7") },
+                        call: "teamsMeeting")
+                        {
+                            PlaceholderImageUri = "https://imageendpoint"
+                        }
+                    }
+                },
+                {
+                    "jack",
+                    new MediaInput() {
+                        Participant = new ParticipantInput(
+                        id: new CommunicationIdentifierModel() { MicrosoftTeamsUser = new MicrosoftTeamsUserIdentifierModel("fa4337b5-f13a-41c5-a34f-f2aa46699b61") },
+                        call: "teamsMeeting")
+                        {
+                            PlaceholderImageUri = "https://imageendpoint"
+                        }
+                    }
+                },
+                 {
+                    "jane",
+                    new MediaInput() {
+                        Participant = new ParticipantInput(
+                        id: new CommunicationIdentifierModel() { MicrosoftTeamsUser = new MicrosoftTeamsUserIdentifierModel("2dd69470-dc25-49cf-b5c3-f562f08bf3b2") },
```

Comment:
In our final model-based API we should use the SDK identifier types and not the identifier model types.

Reply:
Could you clarify what you mean here by the SDK identifier types?

Reply:
So looks like right now, since it's nested inside the ParticipantInput, we won't be able to use the SDK identifier types. However, once we make polymorphic types for the ParticipantInput class, we should be able to switch to using the SDK Identifier Types